examples/evaluate_lightning_text_classification.py

```diff
@@ -0,0 +1,36 @@
+import pprint
+from pathlib import Path
+
+from embeddings.defaults import RESULTS_PATH
+from embeddings.pipeline.lightning_text_classification import TextClassificationPipeline
+
+root: Path = RESULTS_PATH.joinpath("lightning_sequence_classification")
+root.mkdir(parents=True, exist_ok=True)
+
+# pipeline = LightningClassificationPipeline(
+#     embedding_name="allegro/herbert-base-cased",
+#     dataset_name="clarin-pl/cst-wikinews",
+#     input_column_name=["sentence_1", "sentence_2"],
+#     target_column_name="label",
+#     task_train_kwargs={"max_epochs": 1, "gpus": 1},
+#     task_model_kwargs={"pool_strategy": "cls", "learning_rate": 5e-4}
+# )
+
+pipeline = TextClassificationPipeline(
```

Comment:
This example should be also used in form of test. With limited number of epochs and dataset size

Reply:
I wrote test but there is an error with pytorch-lightning and transformers version; it's fixed in transformers > 4.10.x so this PR should fix it https://github.com/CLARIN-PL/embeddings/pull/116
I can append the tests in the PR or in this one if other PR will be merged faster

Reply:
Ok, so I would add tests in the separate PR. Without fixing versions until the versions on the main will be updated.

Reply:
@djaniak paste PR reference when it will be ready